CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ project(hyperon)
 
 include(ExternalProject)
 
+option(GIT "Adds git features to hyperon library; requires OpenSSL" ON)
```

Comment:
You need also add `"-DGIT=${GIT}"` into two `ExternalProject_Add` macroses below. Otherwise `GIT` option is not propagated into the sub-projects. Thus right now making project with `-DGIT=OFF` doesn't affect `hyperonc` and `hyperonpy`.